status/pricing.py

```diff
@@ -38,25 +38,34 @@ def _validate_date_string(self, date):
     # _____________________________ FETCH METHODS _____________________________
 
     def fetch_published_doc_version(self, version=None):
-        db = self.application.cost_calculator_db
         if version is not None:
```

Comment:
What's the purpose of this if-else block? It seems in either case we are just grabbing the first element of the list, so no need to query in two different ways?

Reply:
Ah, makes sense 👍